src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -99,6 +119,8 @@
 public final class PreprocessIntervals extends GATKTool {
     public static final String BIN_LENGTH_LONG_NAME = "bin-length";
     public static final String PADDING_LONG_NAME = "padding";
+    public static final String GRID_LONG_NAME = "grid";
```

Comment:
Since this is a boolean argument and "grid" is not unambiguously a verb, perhaps "use-grid"?  Or "fix-grid", etc.